examples/platforms/cc2538/uart.c

```diff
@@ -54,14 +54,22 @@ static void processTransmit(void);
 static const uint8_t *sTransmitBuffer = NULL;
 static uint16_t sTransmitLength = 0;
 
+typedef struct RecvBuffer
+{
+    uint16_t Offset;
+    uint16_t Length;
+} RecvBuffer;
+
 static uint8_t sReceiveBuffer[kReceiveBufferSize];
```

Comment:
In my testing I increased `kReceiveBufferSize` from 128 to 1024, but I didn't include it in this change. Was there a specific reason 128 was chosen?